leasing/tests/models/test_calculate_invoices.py

```diff
@@ -575,3 +576,136 @@ def test_calculate_invoices_uses_correct_receivable_type(
         assert len(invoice_data) == 1
         assert len(invoice_data[0]) == 1
         assert invoice_data[0][0]["rows"][0]["receivable_type"] == receivable_type
+
+
+@pytest.mark.django_db
+def test_calculate_invoices_uses_override_receivable_type(
+    django_db_setup,
+    service_unit_factory,
+    receivable_type_factory,
+    lease_factory,
+    tenant_factory,
+    contact_factory,
+    tenant_contact_factory,
+    tenant_rent_share_factory,
+    rent_factory,
+    contract_rent_factory,
+):
+    """
+    If an override receivable type if defined in a rent's contract rent, that
+    receivable type is used by all invoice rows in the rent's invoice.
```

Comment:
:thinking:

Reply:
Yes. That is the requirement I have understood from ticket MVJ-550 description and the Slack discussion with Kimmo yesterday at 16:07.

Reply:
>"Laskuja muodostaessa käytetään korvaavaa saamislajia laskulle jos se on määritelty yhdessäkään sopimusvuokrassa. Tämä saamislaji on sama kaikille saman laskun laskuriveille."

Reply:
This does leave room for manual mistake, if the valmistelija 1) adds two contract rents, even though only 1 is ever wanted, and 2) selects different overrides for both.

I guess I could write a custom validator that checks that all contract rents in the same rent have the same override, if it is specified. Not sure how difficult that is to implement across multiple contract rents.